.github/workflows/documentation.yml

```
@@ -0,0 +1,96 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+
+jobs:
+  build:
+    name: Build Documentation
+    runs-on: ubuntu-latest
+
+    if: github.repository_owner == 'sapphiredev'
+    outputs:
+      NAME: ${{ steps.env.outputs.NAME }}
+      TYPE: ${{ steps.env.outputs.TYPE }}
+      SHA: ${{ steps.env.outputs.SHA }}
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - name: Use Node.js v16
+        uses: actions/setup-node@eeb10cff27034e7acf239c5d29f62154018672fd # tag=v3
+        with:
+          node-version: 16
+          cache: yarn
+          registry-url: https://registry.npmjs.org/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Build Documentation
+        run: yarn docs
+      - name: Upload Documentation Artifacts
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # tag=v3
+        with:
+          name: docs
+          path: docs/api.json
+      - name: Set Output
+        id: env
+        run: |
+          echo "::set-output name=NAME::${GITHUB_REF_NAME}"
+          echo "::set-output name=TYPE::${GITHUB_REF_TYPE}"
+          echo "::set-output name=SHA::${GITHUB_SHA}"
+  upload:
+    name: Upload Documentation
+    needs: build
+    runs-on: ubuntu-latest
+    env:
+      NAME: ${{ needs.build.outputs.NAME }}
+      TYPE: ${{ needs.build.outputs.TYPE }}
+      SHA: ${{ needs.build.outputs.SHA }}
+    steps:
+      - name: Checkout Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - name: Use Node.js v16
+        uses: actions/setup-node@eeb10cff27034e7acf239c5d29f62154018672fd # tag=v3
+        with:
+          node-version: 16
+          cache: yarn
+          registry-url: https://registry.npmjs.org/
+      - name: Install Dependencies
+        run: yarn --immutable
+      - name: Download Documentation Artifacts
+        uses: actions/download-artifact@fb598a63ae348fa914e94cd0ff38f362e927b741 # tag=v3
+        with:
+          name: docs
+          path: docs
+      - name: Checkout Documentation Project
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+        with:
+          repository: 'sapphire/docs'
```

```suggestion
          repository: 'sapphiredev/docs'
```

I wish we had that but it's claimed